php/libraries/NDB_Caller.class.inc

```diff
@@ -361,8 +400,12 @@ class NDB_Caller
                 $_REQUEST['sessionID']
             );
         }
-        if (method_exists($menu, 'save')) {
+        if (method_exists($menu, 'save') && self::shouldCallSave()) {
```

Comment:
And the decision to call `save()` is entrusted to `NDB_Caller`. I certainly wouldn't want other classes having the choice of when `save()` should be called. Someone might decide doing it in a `GET` is a good idea!